src/test/fuzz/util.cpp

```diff
@@ -160,6 +160,41 @@ int FuzzedSock::Connect(const sockaddr*, socklen_t) const
     return 0;
 }
 
+int FuzzedSock::Bind(const sockaddr*, socklen_t) const
+{
+    // Have a permanent error at bind_errnos[0] because when the fuzzed data is exhausted
+    // SetFuzzedErrNo() will always return the first element and we want to avoid Bind()
+    // returning -1 and setting errno to EAGAIN repeatedly.
```

Comment:
This comment threw me off a little. Is this clearer to what is actually happening inside `SetFuzzedEerrNo`?

```suggestion
    // Have a permanent error at bind_errnos[0] because when the fuzzed data is exhausted
    // SetFuzzedErrNo() will always set the first element. We want to avoid Bind()
    // returning -1 and setting errno to EAGAIN repeatedly.
```

Reply:
Hmm, I see where the confusion comes from - `SetFuzzedErrNo()` does not _return_ the value but rather sets the global variable `errno` to it.

What about this:

```suggestion
    // Have a permanent error at bind_errnos[0] because when the fuzzed data is exhausted
    // SetFuzzedErrNo() will always set the global errno to bind_errnos[0]. We want to avoid Bind()
    // returning -1 and setting errno to EAGAIN repeatedly because a legit code should retry on
    // temporary errors leading to an infinite loop.
```

Reply:
Thanks, that makes it clearer.

Reply:
Done, thanks!